extensions/ANDROID/EGL_ANDROID_GLES_layers.txt

```diff
@@ -0,0 +1,63 @@
+Name
+
+    ANDROID_GLES_layers
+
+Name Strings
+
+    EGL_ANDROID_GLES_layers
+
+Contributors
+
+    Cody Northrop
+
+Contact
+
+    Cody Northrop, Google LLC (cnorthrop 'at' google.com)
+
+Status
+
+    Draft
+
+Version
+
+    Version 1, March 3, 2019
+
+Number
+
+    EGL Extension #132
+
+Extension Type
+
+    EGL client extension
+
+Dependencies
+
+    Requires EGL 1.5 or EGL_EXT_client_extensions
+
+Overview
+
+    This extension indicates the EGL loader supports GLES layering on Android.
+    It does not add any requirements to drivers or hardware.
+
+    See native/opengl/libs/EGL/GLES_layers.md for more information.
```

Comment:
Let's expand this to `frameworks/native/opengl/libs/EGL/GLES_layers.md` and mention that it is in the Android tree.

Reply:
updated.